java/java.openjdk.project/src/org/netbeans/modules/java/openjdk/project/ActionProviderImpl.java

```diff
@@ -171,8 +171,14 @@ public void invokeAction(String command, Lookup context) throws IllegalArgumentE
             }
         }
         FileObject scriptFO = script;
+        Settings settings = project.getLookup().lookup(Settings.class);
+        Properties props = new Properties();
+        if (settings.isUseAntBuild()) {
```

Comment:
The other `ActionProviderImpl` checks `settings != null` before dereferencing. But a direct dereference was already there in `COMMAND_BUILD_GENERIC_FAST` branch.